src/components/partner/dto/update-partner.dto.ts

```diff
@@ -49,6 +50,14 @@ export abstract class UpdatePartner {
   @NameField({ nullable: true })
   readonly address?: string;
 
+  @Field(() => String, { nullable: true })
+  @Transform(({ value: str }) => (str ? externalUrlWithProtocol(str) : null))
+  readonly websiteUrl?: string;
```

Comment:
For updates, we specifically want to allow `null` type for TS.
`undefined/?` comes from omission from JSON body, which we interrupt (by PATCH convention) to mean "unchanged".
An explicit `null` from input means to clear the current value.
```suggestion
  readonly websiteUrl?: string | null;
```